rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.75.0"
+channel = "1.76.0"
```

Comment:
@daidokoro I had to push this to 1.78 in my last change due to some library upgrade need.  We need to check if its ok with your code ( Guess it will )